dir.props

```diff
@@ -17,7 +17,7 @@
   <!-- Trying to mitigate the number of warnings that the solution contains by
     adding this into our Release builds.  As a result, it should be caught in
     our PRs because of the CI builds -->
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+  <!-- <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
This was commented out because of the [CS1701](https://msdn.microsoft.com/en-us/library/2h4x8b08.aspx) warnings due to the different assembly versions it is resolving.

When building our project, we are encountering ~900 warnings like this:
```
warning CS1701: Assuming assembly reference 'System.Runtime, Version=4.0.20.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Fx.Portability' matches identity 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Runtime', you may need to supply runtime policy [D:\git\conniey\dotnet-apiport\src\ApiPort\ApiPort.csproj]
```
The reason for that is because Microsoft.Fx.Portability targets netstandard1.3 and ApiPort targets netcoreapp1.0.  So at build time, Microsoft.Fx.Portability references System.Runtime 4.0.20.0... but ApiPort uses System.Runtime 4.1.0.0.  This should not result in problems since we are only adding to the API surface area... but there is an issue about this https://github.com/aspnet/Mvc/issues/5050#issuecomment-288931208.  Should I make another in the right location?